### PR TITLE
PMM-5727 Added flag for metrics totals only

### PR DESCRIPTION
**Pull request by percona-csalguero**

### CODE REVIEW

File: services/analytics/object_details.go [hunk 1]
```diff
@@ -67,6 +67,7 @@ func (s *Service) GetMetrics(ctx context.Context, in *qanpb.MetricsRequest) (*qa
 			in.GroupBy,
 			dimensions,
 			labels,
+			in.Totals,
```
Comment (BupycHuk) [inline services/analytics/object_details.go:70]: it's always false :)

File: services/analytics/object_details.go [hunk 2]
```diff
@@ -88,9 +88,10 @@ func (s *Service) GetMetrics(ctx context.Context, in *qanpb.MetricsRequest) (*qa
 		in.GroupBy,
 		dimensions,
 		labels,
+		true, // get Totals
 	)
 	if err != nil {
-		return nil, fmt.Errorf("error in quering totals:%v", err)
+		return nil, errors.Wrapf(err, "cannot get get metrics totals")
```
Comment (BupycHuk) [inline services/analytics/object_details.go:94]: get get?

File: Gopkg.toml [hunk 3]
```diff
@@ -11,7 +11,7 @@ required = [
 
 [[constraint]]
   name = "github.com/percona/pmm"
-  branch = "PMM-2.0"
+  branch = "PMM-5727"
```
Comment (AlekSi) [inline Gopkg.toml:14]: Please don't forget to update it once it is merged

Comment (AlekSi) [reply Gopkg.toml:14]: @percona-csalguero ^ that might be the cause of the problem. Please create a new PR that revert the revert commit, and then fixes this

Comment (AlekSi) [reply Gopkg.toml:14]: /cc @BupycHuk @askomorokhov

Comment (BupycHuk) [reply Gopkg.toml:14]: No, anyway we shouldn't merge this PR, it changes logic.

Comment (AlekSi) [reply Gopkg.toml:14]: It is already merged and reverted :) I'm not saying we should merge a new PR – but we must create it

Comment (BupycHuk) [reply Gopkg.toml:14]: I answered to first sentence about the cause of the problem. I agree with everything else.

**Pull request merged**